src/mo_config/flags.ml

```diff
@@ -24,6 +24,8 @@ let check_ir = ref true
 let package_urls : string M.t ref = ref M.empty
 let actor_aliases : string M.t ref = ref M.empty
 let actor_idl_path : string option ref = ref None
+let max_stable_pages_default = 65536
```

Comment:
Is this abstraction useful? It's not used except on the line that follows, is it?

Reply:
Its used in the compiler flag documentation, to report the default.